crates/ruff/src/checkers/ast/mod.rs

```diff
@@ -4918,7 +4890,14 @@ impl<'a> Checker<'a> {
             if self.enabled(Rule::UnusedImport) {
                 pyflakes::rules::unused_import(self, scope, &mut diagnostics);
             }
-
+            if self.enabled(Rule::UnconventionalImportAlias) {
```

Comment:
Should the function containing this still be names `check_dead_scopes`? i feel it has become more of a `check_deferred_scopes`

Reply:
Going to change this in a separate PR.